src/Esfa.Vacancy.Infrastructure/Services/CreateApprenticeshipSandboxService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
+using Esfa.Vacancy.Domain.Entities;
+using Esfa.Vacancy.Domain.Interfaces;
+using Newtonsoft.Json;
+using SFA.DAS.NLog.Logger;
+
+namespace Esfa.Vacancy.Infrastructure.Services
+{
+    public class CreateApprenticeshipSandboxService : ICreateApprenticeshipService
+    {
+        private readonly ILog _logger;
+
+        public CreateApprenticeshipSandboxService(ILog logger)
+        {
+            _logger = logger;
+        }
+
+        public async Task<int> CreateApprenticeshipAsync(CreateApprenticeshipParameters parameters)
+        {
+            var serialized = JsonConvert.SerializeObject(parameters);
+            _logger.Info($"Sandbox Request for Create Apprenticeship : {serialized}");
+            return await Task.FromResult(0);
```

Comment:
Is it worth returning a fake number rather than 0? Wondering if 0 looks like it didn't work.

Reply:
The response code is going to be 200 Ok, any other code would give an impression that data has been persisted.